Formatting fixes (#3393)
mattermost/docs: source/install/outbound-proxy.rst

``NO_PROXY``
~~~~~~~~~~~~

The ``NO_PROXY`` environment variable can be set to prevent certain requests from going through the proxy, such as to an SSO provider (e.g. GitLab or SAML) or intranet sites that should be accessible in link previews. It can be configured as a set of comma-separated IP addresses (e.g. ``1.2.3.4``), IP address ranges specified in CIDR notation (e.g. ``1.2.3.4/8``), or domain names. An IP address or domain name can also include a port number.

> When specifying a domain name, ensure that domain name also matches its subdomains. A domain name with a leading ``.`` matches only subdomains. For example, ``example.com`` matches both ``example.com`` as well as ``sub.example.com`` while ``.example.com`` only matches the latter.

Reviewer: This isn't quite correct since we're trying to show the difference between using `example.com` and `.example.com`. How about this?
```suggestion
When a domain name is specified, the domain and all of its subdomains are matched, however a domain name with a leading ``.`` only matches the subdomains. For example, ``example.com`` matches both ``example.com`` and ``sub.example.com`` while ``.example.com`` only matches the latter.
```

``NO_PROXY``
~~~~~~~~~~~~

> The ``NO_PROXY`` environment variable can be set to prevent certain requests from going through the proxy, such as to an SSO provider (such as GitLab or SAML) or to intranet sites that should be accessible in link previews. It can be configured as a set of comma-separted IP addresses (e.g. ``1.2.3.4``), IP address ranges specified in CIDR notation (e.g. ``1.2.3.4/8``), or domain names. An IP address or domain name can also include a port number.

Reviewer 2: ```suggestion
The ``NO_PROXY`` environment variable can be set to prevent certain requests from going through the proxy, such as to an SSO provider (e.g. GitLab or SAML) or intranet sites that should be accessible in link previews. It can be configured as a set of comma-separated IP addresses (e.g. ``1.2.3.4``), IP address ranges specified in CIDR notation (e.g. ``1.2.3.4/8``), or domain names. An IP address or domain name can also include a port number.
```
Should we have an example of an IP address or domain name with a port number included?

Author: Idk. I think either way people will get the point.

> When specifying a domain name, that domain name also matches its sub domains as well. A domain name with a leading ``.`` matches only sub domains. For example, ``example.com`` matches both ``example.com`` as well as ``sub.example.com`` while ``.example.com`` only matches the latter.

Reviewer 2: ```suggestion
When specifying a domain name, ensure that domain name also matches its subdomains. A domain name with a leading ``.`` matches only subdomains. For example, ``example.com`` matches both ``example.com`` as well as ``sub.example.com`` while ``.example.com`` only matches the latter.
```
